### ISSUE TITLE
Pluglin broken?

### ISSUE THREAD

**Issue by laxris** [marc0l92/joplin-plugin-plantUML#14]
It seems the plugin fails during load time:

![image](https://user-images.githubusercontent.com/4000627/211120390-0f85b75f-5cef-4882-8044-6ee6c004e7de.png)

Maybe a wrong path?

**Comment by Zeg31** [comment]
Same here. May be the [Pull request](https://github.com/marc0l92/joplin-plugin-plantUML/pull/12) solves it for Linux.

Unfortunately the project is no longer maintained. (see warning at README.md)

**Comment by DiqitalB** [comment]
> Same here. May be the [Pull request](https://github.com/marc0l92/joplin-plugin-plantUML/pull/12) solves it for Linux.
> 
> Unfortunately the project is no longer maintained. (see warning at README.md)

Doesn't fix it, built the src code, doesn't fix the issue.

**Comment by arussu** [comment]
Same here, I get:
```logs
2023-01-11 12:18:33: joplin.plugins: "Uncaught exception in plugin "com.github.marc0l92.joplin-plugin-plantUML":", "Error: ENOENT: no such file or directory, stat '/var/folders/5m/h3q6_fwj2ngfs0v82l8dt78c0000gn/T/joplin-plantUml2-plugin/'
Error: ENOENT: no such file or directory, stat '/var/folders/5m/h3q6_fwj2ngfs0v82l8dt78c0000gn/T/joplin-plantUml2-plugin/'
```
Did anyone find a way to fix this ?

**Comment by marc0l92** [comment]
Thank @samrook for the fix.
I published your changes in the version 1.11

**Comment by Zeg31** [comment]
Works fine for me.

Thank you very much @samrook and @marc0l92.
